src/server/src/main/resources/db/cassandra/025_lighten_load_repair_run_scans.cql

```diff
@@ -0,0 +1,27 @@
+--
+--  Copyright 2020 The Last Pickle Ltd
+--
+--  Licensed under the Apache License, Version 2.0 (the "License");
+--  you may not use this file except in compliance with the License.
+--  You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+--  Unless required by applicable law or agreed to in writing, software
+--  distributed under the License is distributed on an "AS IS" BASIS,
+--  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--  See the License for the specific language governing permissions and
+--  limitations under the License.
+--
+-- Upgrade to handle diagnostic event subscriptions
+
+CREATE TABLE IF NOT EXISTS repair_run_by_cluster_v2 (
+    cluster_name text,
+    id timeuuid,
+    repair_run_state text,
```

Comment:
why not just `alter table add column` on repair_run_by_cluster ?

Reply:
Because I'm also changing the clustering order, to fix #568